cmd/cri-containerd/options/options.go

```diff
@@ -31,12 +31,20 @@ const configFilePathArgName = "config"
 
 // ContainerdConfig contains config related to containerd
 type ContainerdConfig struct {
-	// ContainerdRootDir is the root directory path for containerd.
-	ContainerdRootDir string `toml:"root"`
-	// ContainerdSnapshotter is the snapshotter used by containerd.
-	ContainerdSnapshotter string `toml:"snapshotter"`
-	// ContainerdEndpoint is the containerd endpoint path.
-	ContainerdEndpoint string `toml:"endpoint"`
+	// RootDir is the root directory path for containerd.
+	RootDir string `toml:"containerd_root_dir"`
```

Comment:
Remove `containerd_` and for the other configuration.

run `cri-containerd --default-config` you'll know why. :)